mcdemoaux/vision/vis.py

```diff
@@ -25,3 +25,47 @@ def run(self):
                 cv2.imshow(data[0], data[1])
             cv2.waitKey(300)
 
+class VisualizerMac():
+    def __init__(self):
+        self.queue = deque(maxlen=10)
+
+    def __call__(self, *args):
+        cv2.imshow(args[0], args[1])
+        cv2.waitKey(1)
+
+    def stop(self):
+        pass
+
+    def start(self):
+        pass
+
+    def run(self):
+        pass
+
+class VisualizerBlank():
+    def __init__(self):
+        pass
+
+    def __call__(self, *args):
+        pass
+
+    def start(self):
+        pass
+
+    def stop(self):
+        pass
+
+    def run(self):
+        pass
+
+def Visualizer(blankvis=False):
```

Comment:
I would turn the visualizer off by default on Mac. So, I'd make the parameter `blankvis=None`. If it is `None`, then it is treated as `False` for Windows and Linux, and `True` for Mac. Thus, it will still be possible to explicitly change it to `False` and `True`, but the default behavior will be os-dependent.

Reply:
@Necr0x0Der  i've changed that. Please check if i got you right and now everything here as expected.